### DISCUSSION TITLE
Any pointers on how custom pages are being used?

### DISCUSSION THREAD

**Discussion by ovizii** [mealie-recipes/mealie#309] [Q&A]
I would very much like to see a more detailed explanation on how to use custom pages. 
I have read this but I don't fully understand how and where to use custom pages. https://hay-kot.github.io/mealie/getting-started/organizing-recipes/

**Comment by sephrat** [comment]
Did you check out this doc page? https://hay-kot.github.io/mealie/site-administration/building-pages/
Also, in the next release, categories will be listed by default if you haven't set up any custom page. So basically, custom pages make it possible to virtually merge categories on the sidebar if you have too many of them you want to group.

As stated in the doc, this feature is quite a new one. Feel free to provide feedback and suggest ideas to improve them. I have some on my mind already, still need to work them through.

**Reply by ovizii** [reply]
Ah, I think I get it a little better now. 

So basically a custom page is a page filtered by tags, right? I think this is related to the discussion about how to handle tags. I have already added my thoughts to that discussion. => https://github.com/hay-kot/mealie/discussions/242#discussioncomment-619354

**Reply by sephrat** [reply]
Actually it's filtered by categories, but there's a bug in the current release that makes it look like it's using tags. https://github.com/hay-kot/mealie/issues/290
Check out the demo to see what it will look like in the next release: https://mealie-demo.hay-kot.dev/admin/settings

**Comment by michael-genson** [comment]
Mealie has changed drastically since this discussion was opened, so I'm closing it since it's no longer relevant. I believe the closest thing we have to custom pages is cookbooks, so I would take a look at those.
